app/config/FrontendAppConfig.scala

```diff
@@ -37,7 +37,7 @@ class FrontendAppConfig @Inject() (configuration: Configuration) {
 
   lazy val maxTransportEquipment: Int = configuration.get[Int]("limits.maxTransportEquipment")
 
-  lazy val maxDocuments: Int = configuration.get[Int]("limits.maxDocuments")
+  lazy val maxDocumentsPerType: Int = configuration.get[Int]("limits.maxDocumentsPerType")
```

Comment:
This is a minor suggestion but it it is not out of the realms of possibility that the limit for one of the document types could change in a future DDNTA change. Could we define the limits separately?